autoload/lsp/lsp.vim

```diff
@@ -31,9 +31,13 @@ var ftypeServerMap: dict<list<dict<any>>> = {}
 var lspInitializedOnce = false
 
 def LspInitOnce()
-  prop_type_add('LspTextRef', {highlight: 'Search', override: true})
```

Comment:
could you keep 'override: true' ? though you had done it for 'LspDiagInlineError' etc,
but actually someone perhaps disagree with it, which preferred to had 'override: true' to work with 'cursorline'.

Reply:
It's unfortunate that vim doesn't allow us to override only the cursorline. It might make sense to add another option "propOverride", which when set will set the override flag on all props.

Reply:
If the 'override' attribute is not set for these text properties and the user has set the 'cursorline' option, then the references in the current line will not be highlighted or shown.  Won't that confuse the user?

Reply:
Possible, the alternative is that visual selection / search etc, will highlighted below the prop. So it's either show prop on top of everything, or show cursorline, visual selection, and search above the prop.